src/options.js

```diff
@@ -9,6 +9,7 @@ function baseOption(strictMode) {
             // EN
             new parser.ENISOFormatParser(strictMode),
             new parser.ENDeadlineFormatParser(strictMode),
+            new parser.ENRelativeDateFormatParser(strictMode),
```

Comment:
Relative date format is casual by nature, and should not by be in strict mode in the first place. Instead of supporting strict mode, just don't include the parser into strict configuration is probably better.